src/Compilers/VisualBasic/Portable/Syntax/SyntaxKindFacts.vb

```diff
@@ -793,6 +799,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                    {"istrue", SyntaxKind.IsTrueKeyword},
                    {"join", SyntaxKind.JoinKeyword},
                    {"key", SyntaxKind.KeyKeyword},
+                   {"load", SyntaxKind.LoadKeyword},
```

Comment:
I think we should fix the Scanner then.  It seems really inconsistent to me to have these be contextual keywords.